### DISCUSSION TITLE
How to use selfhosted Renovate, while managed Renovate is enabled for whole GH org ?

### DISCUSSION THREAD

**Discussion by lukasmrtvy** [renovatebot/renovate#17679] [Request Help]
How to use self hosted Renovate, while managed Renovate is enabled for whole GH org ? The problem is if I create renovate.json, managed Renovate will start to use renovate.json file as well as selfhosted Renovate. Is possible to use some variable, like RENOVATE_CONFIG_FILE_NAME=renovate-selfhosted.json to set nonstandard naming, which can be ignored by managed Renovate ? Thanks

**Comment by rarkins** [comment]
So the managed Renovate is not running on the particular repo(s) you want to self host for? Does the repo have a "Configure Renovate" PR already?

**Reply by lukasmrtvy** [reply]
Renovate is enabled for GH org, I don't have permission to disable it for my repository under this GH org. ( Repository is already onboarded by managed Renovate, cuz I tried to use aws-ami datasource via regex manager, seems its not possible to pass AWS creds to aws-sdk via encryption, but that's another story.. ). How can I disable managed Renovate?

**Reply by rarkins** [reply]
You could disable the managed Renovate by adding `"enabled": false` to its `renovate.json`. Then make sure to set requireConfig=ignored yourself as recommended below

**Comment by PhilipAbed** [comment]
use a self hosted global config [config.js](https://docs.renovatebot.com/getting-started/running/#using-configjs)
to ignore the repository config file it is possible by using: [requireConfig: ignored](https://docs.renovatebot.com/self-hosted-configuration/#requireconfig)

**Reply by rarkins** [reply]
Beware to use a different branchPrefix though so that you don't try to use the same branch names as the other Renovate

**Reply by lukasmrtvy** [reply]
I want to use renovate.json in my repository, but only for self-hosted renovate ( while managed Renovate is enabled for my GH organization ), not sure for what is `requireConfig: ignored` required.  Self-hosted Renovate is configured via RENOVATE_CONFIG

**Reply by rarkins** [reply]
You can read the docs for requireConfig to understand it. You or a colleague will need to configure it for the hosted app if you wish for it to ignore your renovate.json

**Reply by rarkins** [reply]
You can read the docs for requireConfig to understand it. You or a colleague will need to configure it for the hosted app if you wish for it to ignore your renovate.json

**Reply by rarkins** [reply]
You can read the docs for requireConfig to understand it. You or a colleague will need to configure it for the hosted app if you wish for it to ignore your renovate.json

**Reply by rarkins** [reply]
You can read the docs for requireConfig to understand it. You or a colleague will need to configure it for the hosted app specifically for this repo if you wish for it to ignore a renovate.json you have added
